### PR TITLE
Bug 1236336 - pages without title show blankspace instead of null

### PR DESCRIPTION
**Pull request by kingroryg**
fix for this bug : https://bugzilla.mozilla.org/show_bug.cgi?id=1236336

Screenshot : 

screen shot 2016-01-12 at 8 36 36 pm

### CODE REVIEW

File: add-on/panels/js/roomViews.jsx [hunk 1]
```diff
@@ -521,6 +521,9 @@ loop.roomViews = (function(mozL10n) {
         var roomTitle = this.state.roomName ||
                         this.state.roomContextUrls[0].description ||
                         this.state.roomContextUrls[0].location;
+        if (typeof roomTitle === "object" && roomTitle === null) {
+          roomTitle = " ";
```
Comment (Standard8) [inline add-on/panels/js/roomViews.jsx]: I think we can change the if statement here to just check `!roomTitle`. There's no need to be more specific.

Also, please can you change what it sets the replacement title to:

```
roomTitle = mozL10n.get("room_name_untitled_page");
```

That should display "Untitled Page" like the room list in the panel. I think that'll be slightly more consistent.

**Comment by kingroryg** [comment]
Made the required changes .

**Comment by Standard8** [comment]
Great, this looks good. Please could you squash the two changesets into one, and change the commit message to:

"Bug 1236336 - If there is no page details, fallback to displaying untitled page rather than null. r=Standard8"

If you could also rebase it on latest master at the same time, that would be useful though not important.

**Comment by kingroryg** [comment]
So should i submit a new PR ?

**Pull request closed without merging**